rust/libdnf-sys/build.rs

```diff
@@ -34,5 +35,17 @@ fn main() -> Result<()> {
         libdnf.display()
     );
     println!("cargo:rustc-link-lib=static=dnf");
+
+    // now, our thin cxx.rs bridge wrapper
+    let mut libdnfcxx = cxx_build::bridge("lib.rs");
+    libdnfcxx
+        .file("cxx/libdnf.cxx")
+        .include("cxx") // this is needed for cxx.rs' `include!("libdnf.hxx")` to work
+        .include("../../libdnf");
+    // until https://github.com/gdesmott/system-deps/pull/32
+    libdnfcxx.includes(libs.iter().flat_map(|lib| lib.1.include_paths.iter()));
```

Comment:
Let's also add at least `libdnfcxx.flag("-std=c++17")` and...OK it looks like warnings are enabled by default.  That's probably good enough for now.

But I bet we could propagate the flags easily enough via prefixing our `cargo build` in the makefile with `env RPMOSTREE_WARNFLAGS=$(warning_flags) cargo build ...` and then parse it here or so.

Reply:
Added `-std=c++17` for now!